modules/atf/util.lua

```diff
@@ -63,12 +63,42 @@ function table.removeKey(t, k)
   return a
 end
 
+local function convertMs(milliseconds)
```

Comment:
@pestOO I'm not sure, that format for Qdatetime will be appropriate for number, that will be given from lua side. If it isn't then all get_dattime should be rewritten and this convertion will be placed in  get_dattime. I don't think this is better then convertion in this file.